packages/grafana-ui/src/components/Table/TableNG/constants.ts

```diff
@@ -14,6 +14,7 @@ export const TABLE = {
   SCROLL_BAR_WIDTH: 8,
   SCROLL_BAR_MARGIN: 2,
   LINE_HEIGHT: 22,
+  HEADER_HEIGHT: 28,
```

Comment:
I took the opportunity to just make this a const and delete some song-and-dance of passing this value around. makes some thing simpler.